{"context": " More than 1 year has passed since last update.SublimeText3\u3067python3\u306ebuild\u74b0\u5883\u3092\u4f5c\u308d\u3046\u3068\u3057\u305f\u6642\uff0cUnicodeEncodeError\u3067\u65e5\u672c\u8a9e\u304c\u8868\u793a\u3067\u304d\u306a\u304b\u3063\u305f\u306e\u3067\u305d\u306e\u89e3\u6c7a\u65b9\u6cd5\u3067\u3059\uff0e\n\n\u74b0\u5883\n\nMac OS X Mavericks v10.9.4\nSublimeText3 Build 3065\nPython 3.4.2\n\n\n\u30a8\u30e9\u30fc\n\u30c7\u30d5\u30a9\u30eb\u30c8\u306e\u72b6\u614b\u3067python3\u306e\u4ee5\u4e0b\u306e\u3088\u3046\u306a\u30b9\u30af\u30ea\u30d7\u30c8\u3092build\u3059\u308b\u3068UnicodeEncodeError\u3068\u306a\u308a\u307e\u3059\uff0e\n\ntmp.py\n# -*- coding: utf-8 -*-\n\nimport platform\n\nprint(platform.python_version())\nprint('\u65e5\u672c\u8a9e')\n\n\n\u30a8\u30e9\u30fc\n3.4.2\nTraceback (most recent call last):\n  File \"/Users/XXXXXXX/Desktop/tmp/tmp.py\", line 6, in <module>\n    print('\\u65e5\\u672c\\u8a9e')\nUnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128)\n[Finished in 0.3s with exit code 1]\n[shell_cmd: python -u \"/Users/XXXXXXX/Desktop/tmp/tmp.py\"]\n[dir: /Users/XXXXXXX/Desktop/tmp]\n[path: /usr/bin:/bin:/usr/sbin:/sbin]\n\n\n\u89e3\u6c7a\u65b9\u6cd5\nTools => Build System => New Build System... \u304b\u3089\u4ee5\u4e0b\u306e\u3088\u3046\u306b\u5165\u529b\u3057\u3066Python3.sublime-build\u3067\u4fdd\u5b58\n\u81ea\u5206\u306fpyenv\u3092\u4f7f\u3063\u3066version\u3092\u5207\u308a\u66ff\u3048\u3066\u3044\u308b\u306e\u3067path\u304c/Users/XXXXXXX/.pyenv/shims/python3\u3067\u3059\u304c\uff0c\u5c0e\u5165\u3057\u3066\u3044\u306a\u3044\u5834\u5408terminal\u304b\u3089which python3\u3067path\u3092\u78ba\u8a8d\u3057\u3066\u5404\u81ea\u66f8\u304d\u63db\u3048\u3066\u4e0b\u3055\u3044\uff0e\n\nPython3.sublime-build\n{\n    \"cmd\": [\"/Users/XXXXXXX/.pyenv/shims/python\", \"-u\", \"$file\"],\n    \"selector\": \"source.python\",\n    \"file_regex\": \"file \\\"(...*?)\\\", line ([0-9]+)\",\n    \"env\": {\"LANG\": \"ja_JP.UTF-8\"}\n}\n\n\nSublimeText3\u3067python3\u306ebuild\u74b0\u5883\u3092\u4f5c\u308d\u3046\u3068\u3057\u305f\u6642\uff0cUnicodeEncodeError\u3067\u65e5\u672c\u8a9e\u304c\u8868\u793a\u3067\u304d\u306a\u304b\u3063\u305f\u306e\u3067\u305d\u306e\u89e3\u6c7a\u65b9\u6cd5\u3067\u3059\uff0e\n\n## \u74b0\u5883\n- Mac OS X Mavericks v10.9.4\n- SublimeText3 Build 3065\n- Python 3.4.2\n\n\n## \u30a8\u30e9\u30fc\n\u30c7\u30d5\u30a9\u30eb\u30c8\u306e\u72b6\u614b\u3067python3\u306e\u4ee5\u4e0b\u306e\u3088\u3046\u306a\u30b9\u30af\u30ea\u30d7\u30c8\u3092build\u3059\u308b\u3068UnicodeEncodeError\u3068\u306a\u308a\u307e\u3059\uff0e\n\n```tmp.py\n# -*- coding: utf-8 -*-\n\nimport platform\n\nprint(platform.python_version())\nprint('\u65e5\u672c\u8a9e')\n```\n\n\u30a8\u30e9\u30fc\n\n```\n3.4.2\nTraceback (most recent call last):\n  File \"/Users/XXXXXXX/Desktop/tmp/tmp.py\", line 6, in <module>\n    print('\\u65e5\\u672c\\u8a9e')\nUnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128)\n[Finished in 0.3s with exit code 1]\n[shell_cmd: python -u \"/Users/XXXXXXX/Desktop/tmp/tmp.py\"]\n[dir: /Users/XXXXXXX/Desktop/tmp]\n[path: /usr/bin:/bin:/usr/sbin:/sbin]\n```\n\n\n## \u89e3\u6c7a\u65b9\u6cd5\nTools => Build System => New Build System... \u304b\u3089\u4ee5\u4e0b\u306e\u3088\u3046\u306b\u5165\u529b\u3057\u3066Python3.sublime-build\u3067\u4fdd\u5b58\n\n\u81ea\u5206\u306fpyenv\u3092\u4f7f\u3063\u3066version\u3092\u5207\u308a\u66ff\u3048\u3066\u3044\u308b\u306e\u3067path\u304c/Users/XXXXXXX/.pyenv/shims/python3\u3067\u3059\u304c\uff0c\u5c0e\u5165\u3057\u3066\u3044\u306a\u3044\u5834\u5408terminal\u304b\u3089which python3\u3067path\u3092\u78ba\u8a8d\u3057\u3066\u5404\u81ea\u66f8\u304d\u63db\u3048\u3066\u4e0b\u3055\u3044\uff0e\n\n```Python3.sublime-build\n{\n\t\"cmd\": [\"/Users/XXXXXXX/.pyenv/shims/python\", \"-u\", \"$file\"],\n\t\"selector\": \"source.python\",\n\t\"file_regex\": \"file \\\"(...*?)\\\", line ([0-9]+)\",\n\t\"env\": {\"LANG\": \"ja_JP.UTF-8\"}\n}\n```\n\n\n\n", "tags": ["Python", "SublimeText"]}